frontend/apps/reader/modules/readerannotation.lua

```diff
@@ -420,7 +420,10 @@ function ReaderAnnotation:addItem(item)
 end
 
 function ReaderAnnotation:onAnnotationsModified(items)
-    items[1].datetime_updated = os.date("%Y-%m-%d %H:%M:%S")
+    local now = os.date("%Y-%m-%d %H:%M:%S")
+    if items[1].datetime ~= now then
```

Comment:
This is a condition that based on the diff looks like it'd nearly always be true? They're likely to be equal in practice?

Reply:
Annotation `datetime` property is filled on creation. Then, when modifying the annotation - adding a note, changing the boundaries etc, the `AnnotationsModified` event is sent and the `datetime_updated` property is set.

Reply:
But, when the annotation is created, the event is sent too, so the chack is false and `datetime_updated` isn't created.

Reply:
Naively I'd expect created and updated to be the same at the time of creation (or put another way, queryable), but I haven't followed the annotation developments in-depth.

Reply:
> But, when the annotation is created, the event is sent too

Feels a bit clumsy to check the datetime against now to guess, in a :onAnnotationsModified(),  that it is not modifed but created.
Isn't there a cleaner way to handle that ? Not calling onAnnotationsModified if it is... not modified ? :) or passing another arg `created=true` ?

Reply:
Done